<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WatcherAdd</class>
 <widget class="QDialog" name="WatcherAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WUTcha editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,10">
     <item row="0" column="1">
      <widget class="QComboBox" name="cbType"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Expression</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leExpression"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOK">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset resource="../xpeccy.qrc">
         <normaloff>:/images/checkbox.png</normaloff>:/images/checkbox.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../xpeccy.qrc"/>
 </resources>
 <connections/>
</ui>
